src/qt/forms/overviewpage.ui

```diff
@@ -115,9 +115,10 @@
              <number>12</number>
             </property>
             <item row="2" column="2">
-             <widget class="QLabel" name="labelWatchPending">
+             <widget class="BitcoinAmountLabel" name="labelWatchPending">
               <property name="font">
                <font>
+                <family>monospace</family>
```

Comment:
Why not just make this the default for `BitcoinAmountLabel`?

Reply:
I've tried different approaches, including
```
const QFont font{"monospace"};
QApplication::setFont(font, "BitcoinAmountLabel");
```

Only the current code works as expected. But I'm still open for suggestions.

Reply:
Done.